src/renderers/native/ReactNativeStackInspector.js

```diff
@@ -58,7 +60,9 @@ if (__DEV__) {
         measure: callback =>
           UIManager.measure(component.getHostNode(), callback),
         props: getHostProps(component),
-        source: component._currentElement && component._currentElement._source,
```

Comment:
Do we need this? It’s only called for items in the `hierarchy` array which I presume are already not `null`s.